WordPress/Classes/ViewRelated/NUX/LoginEpilogueViewController.swift

```diff
@@ -132,6 +150,29 @@ private extension LoginEpilogueViewController {
             topLine.isHidden = true
         }
     }
+
+    func setTableViewMargins() {
+        guard traitCollection.horizontalSizeClass == .regular &&
+            traitCollection.verticalSizeClass == .regular else {
+                tableViewLeadingConstraint.constant = defaultTableViewMargin
+                tableViewTrailingConstraint.constant = defaultTableViewMargin
+                return
+        }
+
+        let marginMultiplier = UIDevice.current.orientation.isLandscape ?
+            TableViewMarginMultipliers.ipadLandscape :
+            TableViewMarginMultipliers.ipadPortrait
+
+        let margin = view.frame.width * marginMultiplier
```

Comment:
If you delete these lines and add this:

```
let isLandscape = UIDevice.current.orientation.isLandscape
        let marginMultiplier = isLandscape ?
        TableViewMarginMultipliers.ipadLandscape :
        TableViewMarginMultipliers.ipadPortrait

        let width = isLandscape ? view.frame.height : view.frame.width

        let margin = width * marginMultiplier
```

it should work.

Reply:
Thanks for the help @mindgraffiti !

Your suggestion helped a lot, but for me getting the `width` still wasn't always correct (I suspect for the exact same reason my approach wasn't always correct - as you stated the view hasn't been updated when this function is called).

> let width = isLandscape ? view.frame.height : view.frame.width

So I determined the width "manually", i.e.: 
```
let screenSizeMax = max(view.frame.width, view.frame.height)
let screenSizeMin = min(view.frame.width, view.frame.height)
let viewWidth = isLandscape ? screenSizeMax : screenSizeMin
```

This worked for me consistently.